### ISSUE TITLE
Is it ok to distribute some of these papers in public?

### ISSUE THREAD

**Issue by kc1212** [blockchain-lab/papers-and-slides#1]
I suspect not all the publishers are happy with open access.

**Comment by qstokkink** [comment]
I agree. Can this be made private or are we out of private repositories as an organisation?

**Comment by devos50** [comment]
We have to pay if we want private repos as an organization. What about replacing the files by links?

**Comment by kc1212** [comment]
Yeah, that should be fine.

On 13 April 2017 20:46:39 CEST, Martijn de Vos <notifications@github.com> wrote:
>We have to pay if we want private repos as an organization. What about
>replacing the files by links?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/blockchain-lab/papers-and-slides/issues/1#issuecomment-293988768

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by synctext** [comment]
[Publication__Horizontal_Scalability_for_blockchains__Aug2017.pdf](https://github.com/blockchain-lab/papers-and-slides/files/1336660/Publication__Horizontal_Scalability_for_blockchains__Aug2017.pdf)
